### ISSUE TITLE
[UI] Focus item bar background doesn't show in builder due to double ## in colour reference

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#13961]
### Mautic Version

5.0.x series

### Way of installing

I cloned the Mautic repo (not recommended or supported)

### PHP version

8.3

### What happened?

When testing another PR I noticed that the bar doesn't show the right colour when creating a focus item:

![screenshot-nimbusweb me-2024 07 15-17_55_44](https://github.com/user-attachments/assets/e4b46475-e9e3-4be7-a7a3-e6348097dc69)

Looks like there's a double hash mark in front of the colour hex reference, see highlight

### How can we reproduce this issue?

Step 1: Create a focus item
Step 2: Select Bar and configure some text to show
Step 3: Notice that the bar should by default be purple, but it's not showing up.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>
